.github/workflows/make-test.yml

```diff
@@ -76,13 +76,18 @@ jobs:
             cxxflags: -Werror
             ctest_args: '-E "qa_polar_..coder_(sc_)?systematic"'
             # ldpath:
-          - distro: 'Fedora 37 (with 0xFE memory initialization)'
-            containerid: 'gnuradio/ci:fedora-37-3.9'
-            cxxflags: -Werror -ftrivial-auto-var-init=pattern
+          - distro: 'Fedora 38'
+            containerid: 'gnuradio/ci:fedora-38-3.10'
+            cxxflags: -Werror
+            ctest_args: '-E "" -j4 --test-load 3'
```

Comment:
This may have saved a few seconds (127 sec vs 150 or 160 sec) but not worth having here for that reason.